### DISCUSSION TITLE
Feature request: option to change ride name color

### DISCUSSION THREAD

**Discussion by ghost** [OpenRCT2/OpenRCT2#23080] [Feature requests]
I would like to request an option to change ride name color

![afbeelding](https://github.com/user-attachments/assets/47e818f8-04dc-4813-972c-eff7d8ba83f9)

**Comment by karst** [comment]
I believe this is already possible thru the usage of Plugins, but it's a bit convoluted, so I agree that we should probably have a more official way of changing the colour.

**Reply by ghost** [reply]
yes, you can put in the window of the ride properties , in the paint tab:

![afbeelding](https://github.com/user-attachments/assets/51ea26f7-9de7-44eb-85a4-85f1ce67e1a2)

**Comment by 733737** [comment]
right now you just put {RED} in front of some letters or {YELLOW} also works with green and some other colors
